src/ADPCMSoundLoader.js

```diff
@@ -1,19 +1,24 @@
-/*
-
-ADPCMSoundLoader loads wav files that have been compressed with the ADPCM format
-
-based on code from Scratch-Flash:
-https://github.com/LLK/scratch-flash/blob/master/src/sound/WAVFile.as
-
-*/
-
 var ArrayBufferStream = require('./ArrayBufferStream');
 var Tone = require('tone');
 var log = require('./log');
 
+/*
```

Comment:
This only has one `*` whereas other multiline comments here seem to start `/**`.